app/terminal/terminal.tsx

```diff
@@ -103,7 +103,7 @@ export default class TerminalComponent extends React.Component<TerminalProps, St
   componentDidMount() {
     this.initialScrollToEnd();
     window.addEventListener("keydown", (this.windowKeyDownListener = this.onWindowKeyDown.bind(this)));
-    window.addEventListener("fullscreenchange", (this.fullScreenListener = () => this.forceUpdate.bind(this)));
+    window.addEventListener("fullscreenchange", (this.fullScreenListener = () => this.forceUpdate()));
     window.addEventListener("resize", (this.resizeListener = this.updateLineLengthLimit.bind(this)));
```

Comment:
nit: maybe use arrow functions instead of `bind` in the other listeners too? just for consistency